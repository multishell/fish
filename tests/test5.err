No matches for wildcard '*ee*'.  (Tip: empty matches are allowed in 'set', 'count', 'for'.)
fish: 	case *ee*
      	     ^
