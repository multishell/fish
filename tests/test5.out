Test 1 pass
Test 2 pass
Test 3 pass
/tmp/fish_wildcard_permissions_test/noaccess/yesaccess/alpha.txt /tmp/fish_wildcard_permissions_test/noaccess/yesaccess/beta.txt /tmp/fish_wildcard_permissions_test/noaccess/yesaccess/delta.txt
